Dosdp support

dosumis opened the pull request.

dosumis commented:
@matentzn - updated README on this branch.  Please give it a test drive.

dosumis commented:
@matentzn - Fixed a couple more issues on this branch.  Should now be working as per updated README.md. Please test.

matentzn commented:
@cmungall seems to me the reason why this fails is because robot does not recognise the --catalog parameters. Ideas?

cmungall commented:
Currently the docker container is built from robot master. Last release to
docker hub a week ago. Probably preceded addition of catalog. We can make a
new build but would like to start tracking formal robot releases soon

On Sun, Jul 29, 2018, 07:47 Nico Matentzoglu <notifications@github.com>
wrote:

> @cmungall <https://github.com/cmungall> seems to me the reason why this
> fails is because robot does not recognise the --catalog parameters. Ideas?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/INCATools/ontology-development-kit/pull/62#issuecomment-408682695>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AADGOW4wpdCaLoyxSGki6Pj4FUkVkVYvks5uLcsPgaJpZM4VaKUt>
> .
>

matentzn commented:
Thanks! Is this build something I can trigger, or will you do it?

matentzn commented:
I rebuilt it oskfull locally, and everything seems to works fine now. In order for the ODK to work properly, someone needs to rebuilt and push the docker oskfull to dockerhub. :) Thanks!

cmungall commented:
Hmm, you show up as an owner here 
https://hub.docker.com/u/obolibrary/dashboard/teams/?team=owners

matentzn commented:
Mea culpa. Pushing now.

matentzn commented:
After updating the oskfull on dockerhub, the travis job completes again.

matentzn commented on template/src/ontology/foobar-edit.owl:
```diff
@@ -7,6 +7,7 @@ Prefix(rdfs:=<http://www.w3.org/2000/01/rdf-schema#>)
 
 Ontology(<http://purl.obolibrary.org/obo/foobar.owl>
 Import(<http://purl.obolibrary.org/obo/foobar/imports/MY-IMPORTED_import.owl>)
+Import(<http://purl.obolibrary.org/obo/foobar/patterns/definitions.owl>)
```
Not too sure whether this is the best way to do this. We should use a robot command to add the import, or only merge the definitions.owl into the final release. There will still be users of the ODK that do not want to use patterns.

dosumis replied:
OK. I'll remove.  We can add a note to the README about adding the import manually if using patterns.

dosumis commented:
If no objections, I'll merge this pull request once i've:
1. Switched the default dosdp.yaml & tsv to something trivial
2. Removed the definitions.owl import from the editor's file.
